apps/mobile/src/components/Community/CommunityMeta.tsx

```diff
@@ -183,20 +183,18 @@ export function CommunityMeta({ communityRef, queryRef }: Props) {
             eventName="Community PFP Press"
             eventContext={contexts.Community}
           />
-          <LinkableAddress
-            chainAddressRef={community.contractAddress}
-            textStyle={{ color: colorScheme === 'light' ? colors.black[800] : colors.offWhite }}
-            font={{ family: 'ABCDiatype', weight: 'Bold' }}
-            eventElementId="Community Contract Address"
-            eventName="Community Contract Address Press"
-            eventContext={contexts.Community}
-          />
+          <View>
```

Comment:
nit: does this need to be wrapped in a View?

Reply:
yeah, I added it because the space-x-1 from the parent's styling wasn't working without it